docs/Contract Creators/l2-sequencer-flag.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: "L2 Sequencer Health Flag Docs"
+permalink: "docs/l2-sequencer-flag/"
+---
+
+The idea behind an Optimistic Rollup (OR) type of protocol is to move all execution off-chain and keep all transaction data available on-chain. Such protocols have a special off-chain component, a Sequencer, that executes and rolls up the Layer 2 transactions by batching multiple transactions into a single one.
```

Comment:
Can we link the word "a Sequencer" to something else so folks can read more?

Reply:
I haven't found a good resource so I attached [the following post](https://medium.com/stakefish/optimistic-rollups-how-they-work-and-why-they-matter-3f677a504fcf). Let me know if this works